runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsConfig.java

```diff
@@ -60,4 +63,26 @@ default boolean enableRestorationBars()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		position = 4,
+		keyName = "toggleRestorationBars",
+		name = "Toggle to hide when not in combat",
+		description = "Visually hides the Status Bars when player is out of combat."
+	)
+	default boolean toggleRestorationBars()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 5,
+		keyName = "hideStatusBarDelay",
+		name = "Delay (seconds)",
```

Comment:
Both of the added configs could be condensed into one `int` config item which does not hide the bars if its value is 0.

Reply:
Also, now that `Units` have been added for config items, this should be annotated with the `Seconds` unit.